docs/roadmap.md

```diff
@@ -0,0 +1,33 @@
+# Roadmap
+
+This roadmap will explain the process of each major release for Cloud 2. As it's still early, this roadmap
+can be subject to change without prior notice.
+
+!!! info
+    From this point, there will be no wiping of any database. Meaning all applications created from now, will exist when we launch Vapor Cloud 2.
+
+    When launching Cloud 2, if there are any duplicate apps between Cloud 1 and Cloud 2, the ones from Cloud 2 will be renamed to `<slug>-v2`
+
+## Alpha (Launched)
+
+The Alpha of Vapor Cloud 2, will serve as an experimental of new features, as we close in on how the release will be. We plan on launching as many of the features needed, in an early stage. More advanced features will come during the Beta phase.
+
+## Beta (Q4 2018)
+
+During the Beta phase, most of the API and features will be locked, here we will also have full backups of all parts of the system, in case of emergency.
+During beta, we will also be enabling billing, however we will be temporary suspending some of the free limitations, in order to let people get going.
```

Comment:
Will we actually be charging people for paid products or just simulating charges?

Reply:
What do you think? .. I think for things like database servers, we should be charging for it, since those take quite a lot of memory

Reply:
It states during beta, so could be "at some point" during billing, maybe after our disaster recovery tests or something?